ci/cluster-config.yaml

```diff
@@ -1,5 +1,6 @@
 tenants:
   - default
+custom_dns_tld: opstracegcp.com
```

Comment:
note to self: I learned that this will be the custom DNS name for the to-be-deployed Opstrace instance, so the current parameter name is bogus. It may be a TLD, but it may also be something like myinstance.mycompany.com -- it's after all a DNS name, and maybe has to be a fully qualified one.